src/beneuro_data/conversion/anipose_interface.py

```diff
@@ -102,6 +118,25 @@ def add_to_nwbfile(
             )
             keypoint_series_objects.append(keypoint_series)
 
+        for angle_name in self.angles:
+            angle_array = self.pose_data[[f"{angle_name}"]].to_numpy()
+            angle_series = PoseEstimationSeries(
+                name=angle_name,
+                data=np.concatenate(
+                    (angle_array, np.zeros((angle_array.shape[0], 1))),
+                    axis=1
+                ),
+                description='Angle information. Second dimension is nans.',
```

Comment:
The converter expects here either 2d or 3d info, but since angles are 1d time series genji told me we can just fill the second dimension with zeros. (I dunno why I wrote nans but i will do another commit with some updates)

Reply:
Have you tried with a 1D array or reshaping to 2D with `(n_timepoints, 1)`?
I guess it might not work if `PoseEstimationSeries` expects (x, y) or (x, y, z). If that's the case, we can use this hack for now and maybe ask the [ndx-pose](https://github.com/rly/ndx-pose) people what they suggest.